### DISCUSSION TITLE
Shadcn UI for React-Native Expo

### DISCUSSION THREAD

**Discussion by bolajithe2nd** [shadcn-ui/ui#4931] [Ideas]
Shadcn UI would be wonderful to use with react native. It is already one of the most used ui library today. It only makes sense to implement it in react native too.

**Comment by cemo** [comment]
@shadcn I think this is great idea. After your last released components I am almost sure that dominance of Shadcn UI will be more and more dominant on React land. I guess it might be possible to create a **high level spec** related to components and it can be implemented by Web and Native. Web part is already implemented by you and native part can be handled by community.
